widgetssdk/src/main/java/com/glia/widgets/call/CallState.java

```diff
@@ -328,16 +331,28 @@ public CallState setTransferring() {
                 .createCallState();
     }
 
-    public boolean isMuteButtonEnabled() {
-        return (isAudioCall() || isVideoCall()) && !showOnHold();
+    public ViewState getGetMuteButtonViewState() {
+        if (isCallVisualizer) {
+            return ViewState.HIDE;
```

Comment:
I might be mistaken about something but in design mocks these buttons are actually disabled (grayed out) in the case of Call Visualizer and not hidden. But based on logic in `applyViewState` if I understood it correctly it actually set them as not visible.

Reply:
Same for the 'speaker' button

Reply:
Also, this is my mistake for not having a clear description of the ticket. But the same logic should also apply to the 'chat' button as well.

Reply:
Made changes based on this thread:

https://salemove.slack.com/archives/C04BTHLDL9Y/p1677679060595919